charts/ingress-nginx/templates/admission-webhooks/job-patch/networkpolicy.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
```

Comment:
Can you make this optional, not everyone is going to have networkpolicy and this would break those who do.

Reply:
similar to this https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/templates/controller-webhooks-networkpolicy.yaml

Reply:
Is it fine to re-use the `controller.admissionWebhooks.networkPolicyEnabled` key or should I put it into the  `controller.admissionWebhooks.patch`?

Reply:
reuse is fine to me.

Reply:
Done!